modules/integration/src/test/scala/scala/cli/integration/TestUtil.scala

```diff
@@ -291,4 +292,12 @@ object TestUtil {
     }
     Await.result(readLineF, timeout)
   }
+
+  def normalizeConsoleOutput(text: String) = {
```

Comment:
NIT This is essentially a duplicate of the functionality from `scala.build.internal.util.ConsoleUtils`... I wonder if it'd be worth the hassle to extract it to some common module, so that we can reuse it in tests.
can also leave it as is though. 🤷‍♂️